website/docs/release_notes.md

```diff
@@ -7,6 +7,103 @@ import ReactPlayer from 'react-player'
 
 # Release notes
 
+## [v1.0.3](https://github.com/VirtusLab/scala-cli/releases/tag/v1.0.3)
+
+## What's new
+
+## Access project configuration with the new `BuildInfo`
+
+`BuildInfo` access your project's build configuration within your Scala code. This feature automatically gathers and generates build information about your project, making project details instantly accessible at runtime.
+
+To generate BuildInfo, either use the `--build-info` command line option or include the `//> using buildInfo` directive in your code.
+
+Upon activation, a `BuildInfo` object becomes accessible on your project's classpath. To use it, simply add the following import into your code:
+
+```
+import scala.cli.build.BuildInfo
+```
+
+This `BuildInfo` object encapsulates information such as the Scala version used, target platform, main class, scalac options, dependencies, and much more for both Main and Test scopes. The generation ensures up-to-date configuration data from both the console options and using directives in your project's sources.
+
+Added by [@MaciejG604](https://github.com/MaciejG604) in [#2249](https://github.com/VirtusLab/scala-cli/pull/2249).
+
+## CompileOnly Dependencies
+
+Now, users can declare dependencies that are exclusively included at the compile time. These dependencies are added to the classpath during compilation, but won't be included when the application is run, keeping your runtime environment lightweight.
+
+To declare such a dependency:
+
+1.  Via the using directive:
+```
+//> using compileOnly.dep "com.github.plokhotnyuk.jsoniter-scala::jsoniter-scala-macros:2.23.2"
+```
+2. Via the command line:
+```
+scala-cli Hello.scala --compile-dep "com.github.plokhotnyuk.jsoniter-scala::jsoniter-scala-macros:2.23.2"
+```
+
+Added by @alexarchambault and [@lwronski](https://github.com/lwronski) in [#2299](https://github.com/VirtusLab/scala-cli/pull/2299), Thanks!
+
+
+## Set globally Java properties
+
+Scala CLI allows users to globally set Java properties for its launcher using the `config` command. This will simplify the JVM properties management process, eliminating the need to pass these properties with each `scala-cli` execution.
+
+To set global Java properties execute the following command:
+
+```
+scala-cli config java.properties Djavax.net.ssl.trustStore=cacerts Dfoo=bar2
+```
+
+When modifying Java properties, remember that you must redefine all of them. It's not possible to update just a single property. Essentially, each time you use the `config` command for Java properties, you replace the entire list of properties.
```

Comment:
Also add something along the lines of:

```
Whenever overwriting existing properties Scala CLI will let you know what was the previous value and in interactive mode ensure that you are ok with replacing them
```

Reply:
It's make sense. I updated release notes.